src/main/main.c

```diff
@@ -80,11 +82,16 @@ static t_error_code	internal_flow(char **envp, t_read_input read_func)
 
 int	main(int argc, char **argv, char **envp)
 {
-	(void)argv;
-	if (1 > argc)
+	t_error_code	err;
+	int				fd;
+
+	if (1 < argc)
 	{
-		printf("Can't Handle files\n");
-		return (ERROR);
+		fd = open(*(argv + 1), O_RDONLY);
```

Comment:
open() can fail
add a printf() msg that indicates what was the error
it does not need to be pretty, i'll replace it when ErrorClass will work